CodeEdit/Features/Editor/Views/EditorView.swift

```diff
@@ -20,35 +20,29 @@ struct EditorView: View {
 
     @EnvironmentObject private var editorManager: EditorManager
 
+    var editorInsetAmount: Double {
+        (EditorTabBarView.height + 1)
+        + (showEditorPathBar
+           ? (EditorPathBarView.height + 1)
+           : 0
+        )
+    }
```

Comment:
This is a little hard to read. The formatting causes the ternary statement to become difficult to follow. I think doing it this way would be better:

```swift
var editorInsetAmount: Double {
    let tabBarHeight = EditorTabBarView.height + 1
    let pathBarHeight = showEditorPathBar ? (EditorPathBarView.height + 1) : 0
    return tabBarHeight + pathBarHeight
}
```

Reply:
Agreed, this is much more readable.